app/services/config.js

```diff
@@ -1,6 +1,7 @@
 const axios = require('axios');
+const { BASE_URL_TIMEOUT } = require('./constants');
```

Comment:
`constants.js ` should go  at the same level of `services` directory not inside of it.